benchmark/Arion_Agent_Benchmark.MD

```diff
@@ -0,0 +1,35 @@
+# ArionAgent performance
+## Overview
+
+This is the performance test of Arion Agent working with ebpf/XDP as downstream programming module and ArionMaster grpc streaming server as upstream metadata source as an entire system.
+
+
+## Test environment
+
+This test is between 2 machines of the same lab, they don't belong to the same IP range (means not located on the same rack but same data center):
+    
+    1. Arion master server
+    
+    2. Arion agent (launched on the same machine of Arion Wing which is XDP as gateway network functionality)
+
+
+## Test workflow and scenario #1 - watch from Arion Master only latency
+
+Latency = ArionAgent finishes receiving N number of neighbors - start to receive Grpc neighbors time (right after watch call to ArionMaster Grpc server finished)
+
+Watch performance from ArionMaster:
+    
+		Watch 5 neighbors performance: 31 us
+    
+		Watch 100k neighbors performance: 379,270 us = ~380 ms
```

Comment:
Also, could we add one section, named test program pointing to the test script(s)?

Reply:
Will put in a test branch, but since code was changed (between PROD and TEST branch) there's no script to control it in MASTER branch